website/content/docs/connect/intentions/intentions.mdx

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
we should name this file index so the url is: `/consul/docs/connect/intentions` instead of `/consul/docs/connect/intentions/intentions`